### DISCUSSION TITLE
Deleting wrong entry

### DISCUSSION THREAD

**Discussion by composertech** [YOURLS/YOURLS#3399] [Q&A]
Hi,
I observed the following behaviour. I added several new URLs through the interface. In the same session I then clicked the delete icon beside an existing link. Result: the wrong URL was deleted. After refreshing the page, the behaviour is correct. Can anyone else verify this? Thanks.

**Comment by ozh** [comment]
Indeed. I made an issue: https://github.com/YOURLS/YOURLS/issues/3400
